app/templates/tools/includes/list.html

```diff
@@ -39,34 +39,28 @@ <h2 class="heading-medium">Your tools</h2>
       {% endfor %}
     {% endif %}
 
-    {% set showRstudio = true %}
-    {% for tool in tools %}
-      {% if tool.app_label == 'rstudio' %}
-        {% set showRstudio = false %}
-      {% endif %}
-    {% endfor %}
-    {% if showRstudio %}
 
+    {% for deployable_tool in deployable_tools %}
     <tr>
-      <td>RStudio</td>
+      <td>{{ deployable_tool.name }}</td>
       <td>
-        {% if rstudio_is_deploying %}
-          Deploying
+        {% if is_deploying[deployable_tool.name] %}
+        Deploying
```

Comment:
I personally prefer to keep the indentation in a way that make sense to has rather than the browser. Browsers don't care while lack of indentation could cause confusion or bugs when human edits this. 

That's my personal preference tho, not a problem or anything :)